### PR TITLE
Feature/default items quantity in feed

### LINKED ISSUE #103: Update RSS feed to support more than 10 episodes
Hi

On my Website I have and podcast with well over 50 Episodes.
But only the last 10 Episodes are in the RSS Feed.
Can I somehow chance something so all or more Episodes are in the RSS Feed?

Maybe you can add this as a option for everybody.

### PR DESCRIPTION
**Pull request by pabamato**
### Description of the Change
 - Change main hook from `wp`to `parse_query ` to allow query modification
 - Filter the feed query and update the `posts_per_rss` value to 250
 - Add custom filter to allow value override 
 - Add block missing dependency
 - Use 10up-Default used on pre-commit
 - Add .phpcs.xml.dist file

### Verification Process

- Create a custom feed
- Add 80 episodes to the feed
- Check how many items the feed includes
- Add filter to the theme to set the feed items to a lower number
- Check feed and confirm has as many items as defined on the theme

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

https://github.com/10up/simple-podcasting/issues/103

### Changelog Entry

- Added
Filter `10up_podcasting_episodes_per_page`

- Changed
Default items on plugin's feeds to 250

- Fixed
'podcast' block core dependency

### CODE REVIEW

File: includes/customize-feed.php [hunk 1]
```diff
@@ -339,3 +339,24 @@ function empty_rss_excerpt( $output ) {
...
+ * @param WP_Query $query The WP_Query instance.
+ * @return void
+ */
+function pre_get_posts( $query ) {
+	// do nothing if not the main query.
+	if ( ! $query->is_main_query() ) {
+		return;
+	}
+
+	// Checks run before include current file so is safe to assume we are on the feed query.
```
Comment (dinhtungdu) [inline includes/customize-feed.php]: Why don't we use `$query->is_feed()` here?

Comment (jeffpaul) [reply includes/customize-feed.php]: @pabamato see ^

Comment (pabamato) [reply includes/customize-feed.php]: @dinhtungdu thanks for the review.
You are correct, I've updated the code to use `$query->is_feed()`.
See: https://github.com/10up/simple-podcasting/pull/109/commits/82194fb6034faafc3adea9fb8d6a4de19249cffc

**Review by dinhtungdu — APPROVED** [review]
LGTM 👍🏽.

Note that I changed the filter and removed the helper because we only use it one time for now.

**Pull request merged**